src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -240,19 +243,23 @@ The `EvmNode` object includes the following items for a given coin or token:
 
 
 ### CounterpartyChannelConfig
+<DevComment>
+  What's the default for `allow_outbound_0conf`?
+</DevComment>
```

Comment:
Sorry for the confusion by writing  `allow_outbound_0conf` as `trust_own_funding_0conf` (the name used in rust-lightning library) here https://github.com/KomodoPlatform/komodo-docs-mdx/pull/31#discussion_r1196799793, please remove `trust_own_funding_0conf` from the docs.

`allow_outbound_0conf`: Optional, defaults to `true`. When setting an outbound channel, it can be used straight away [without waiting].

Reply:
thanks. I removed refs to `trust_own_funding_0conf`, though left the link to 
https://docs.rs/lightning/latest/lightning/util/config/struct.ChannelHandshakeLimits.html#structfield.trust_own_funding_0conf
it in the description

Reply:
Great, thanks :)